### ISSUE TITLE
beforeunload listener not working in Chrome

### ISSUE THREAD

**Issue by L235** [GeneralNotability/spihelper#60]
Frequently I accidentally close the tab where I've got spihelper open (and lost a lot of work), so it'd be good if it prompts me for confirmation to close the tab whenever I've got spihelper open.

**Comment by GeneralNotability** [comment]
@L235 if you have the "action" view open (the view that variously has block options, comments, etc.), it should always prompt for confirmation when you close the tab. Is that not the behavior you're seeing?

**Comment by L235** [comment]
This is very much not the behavior I see.

Steps to replicate (for me):

1. Open any SPI
2. Select an action (say, "Note/comment")
3. Type something into the box
4. Ctrl-W / press close tab

Expected behavior apparently (this has never happened for me): popup asking for confirmation to close tab
Actual behavior: tab is closed, all work is lost

Tested both on my normal browser and on a Chrome "Guest" profile (which comes without extensions, site rules, etc.) with the same result. I also tested it while logged out on Chrome and after running `importScript('User:GeneralNotability/spihelper.js')` in the browser console with the same result.
